.github/workflows/stale_pr.yml

```diff
@@ -0,0 +1,33 @@
+name: Close stale PR
+# This action will add a `stale` label and close immediately every PR that meets the following conditions:
+#   - it is already marked with "merge conflict" label
+#   - the "merge conflict" label was added more than 30 days before.
+
+on:
+  schedule:
+    - cron: '0 10 * * *'
+  workflow_dispatch:
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v7.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          # Do not mark a PR as stale
+          days-before-stale: -1
+          days-before-pr-stale: 30
+          days-before-pr-close: 0
+          # only run the action on merge conflict PR
+          any-of-labels: 'Merge Conflicts'
+          exempt-pr-labels: 'internal, never-stale, ON HOLD, WIP'
```

Comment:
```suggestion
          exempt-pr-labels: 'internal, never-stale, hold, WIP'
```

Reply:
Actually, I want to rename the `hold` label to `ON HOLD`.

Reply:
Done.

There was only 2 closed dependabot PRs marked with `hold`. They are using `ON HOLD` now.